faunadb/tags.go

```diff
@@ -13,3 +18,57 @@ func fieldName(field reflect.StructField) string {
 
 	return name
 }
+
+// parseTag interprets fauna struct field tags
+func parseTag(field reflect.StructField) (name string, ignore, omitempty bool, err error) {
```

Comment:
You don't seem to use the named returns in this function.

Reply:
Do you want me to use `return someStruct, err` or I missed smth ?

Reply:
I meant that you have declared named returns but, you always do explicit returns. That a bit confusing in my opinion.

We usually do this:

```go
func foo() (bar string) {
  bar = "baz"
  return
}
```

or this:

```go
func foo() string {
  return "baz"
}
```

Not both:

```go
func foo() (bar string) {
  bar = "baz"
  return bar
}
```

I hope that's helpful.